thingiverse/ThingiverseService.py

```diff
@@ -78,13 +99,29 @@ def openSettings(self) -> None:
             return
         self._extension.showSettingsWindow()
 
+    @pyqtProperty(list, notify=apiClientChanged)
+    def disabledViews(self) -> list:
+        """
+        Get any disabled views for the current API Client
+        :return: List of view labels that should not be visible
+        """
+        return self._api_client.disabled_views
+
+    @pyqtProperty(str, notify=userNameChanged)
+    def thingiverseUserName(self) -> str:
```

Comment:
Somehow I'd like to unify these two username methods.

Reply:
Maybe we should move it to the ApiClient implementations (let them store their own username) and then just call the right client here (just like we do with all the other methods). That would keep all the service-specific things together.